src/point.class.ts

```diff
@@ -1,344 +1,348 @@
-//@ts-nocheck
-(function(global) {
-  /* Adaptation of work of Kevin Lindsey (kevin@kevlindev.com) */
+import { fabric } from '../HEADER';
 
-  var fabric = global.fabric || (global.fabric = { });
+interface IPoint {
+  x: number
+  y: number
+}
 
-  fabric.Point = Point;
+/**
+ * Adaptation of work of Kevin Lindsey(kevin@kevlindev.com)
+ */
+export class Point {
+
+  x: number
+
+  y: number
+
+  type = 'point'
+
+  static toPoint(from: IPoint) {
+    return new Point(from.x, from.y);
+  }
+
```

Comment:
is this new? what is it about?

Reply:
When @ShaMan123 says `nothing interesting` he is hiding a new motehod.

Reply:
Can't we have the class that can initialized both from 2 numbers or one object?
Is there a multiple pattern calls for constructors?

Reply:
good point!
on it

Reply:
> When @ShaMan123 says `nothing interesting` he is hiding a new motehod.

Haha